Makefile

```diff
@@ -1,5 +1,5 @@
 SKIP_SQUASH?=0
-VERSIONS="1"
+VERSIONS="1 slave-base slave-maven slave-nodejs"
```

Comment:
roughly yes, but we wouldn't change "1" to "master" because at some point in the future we will have both "1" and "2" for master images.

But for the slaves, we aren't going to have multiple versions, so i felt (sort of) comfortable not including a version number in the directory names.

We could rename "1" to "master-1" but those directory names feed heavily into our build infrastructure in terms of dictating how the image ultimately gets named when tagged, so it's easier to leave it as "1".  (the image names become "jenkins-\<directoryname\>-centos7"). 

I also considered a whole new repo for the slave images to avoid this confusion, but it felt like overkill.  I admit we may live to regret this choice but I think we can always separate it out later if that proves the better course of action.